### DISCUSSION TITLE
丹麥科學家探索以 LOVOT 機器人協助照顧失智症患者

### DISCUSSION THREAD

**Discussion by ching020721** [1101-GS4719-C/Python-Community-GS4719-C#772] [科技新知]
LOVOT是LOVE與Robot兩字結合，寵物機器人可以撫慰人心照顧患者，透過丹麥科學家的研究計畫認為LOVOT可為輔助失智患者照護的工具，在現代社會對患者與家人都是能有妥善幫助的。

邁入老年化的社會，經常在醫院看到人手不足，醫護人員焦頭爛額的情況，因此我認為機器人應用在醫療病患上是值得不斷推出精進的，這不僅僅可以適用於文章中失智症病例上，但這個報導卻同時也給我一些想法，包括未來如果我的親人罹患該疾病我會想要使用這項機器人輔助嗎？或是如果選擇要用，那我對於這些機器人會有甚麼疑慮？即使擁有這些疑慮，我仍希望科技能夠應用於這些醫療上，這對於現代社會畢竟是一大的幫助，持續的更進相信對全世界都能夠很好的得到幫助。

資料來源：https://technews.tw/2022/10/09/can-caring-for-a-pet-help-a-person-with-dementia/

**Comment by williamyen0623** [comment]
感覺我晚年會用到，感謝分享，馬上訂一台

**Comment by moonnight3503** [comment]
照顧失智症的家屬本身都會有不少壓力，如果能有AI機器人幫忙照顧的話，感覺能減輕不少負擔。

**Comment by Yunhans** [comment]
之前sony也有出一款機器狗的樣子

**Comment by Pietra1226** [comment]
很高興科技被運用在社會照護上面，這也是我想為科技發展盡一份心力的動力

**Comment by danwan1027** [comment]
冷漠的科技也是能達到溫暖社會的力量

**Comment by StevenWanglolz** [comment]
也許未來這種科技可以解決長者無人照顧的問題，也可以降低照護人員的負擔!

**Comment by wenchi9023** [comment]
能利用科技減少家屬的負擔固然是好，但像原po說的，換作我們是病患親屬，我們真的願意讓機器人來照顧病患嗎? 如果只是單純用來陪伴，我覺得會有不錯的效果，但如果涉及到「照顧」，則不管安全或生活起居上我還是會有些疑慮~

**Comment by Evan0979** [comment]
機器人協助照顧失智症患者感覺很棒!再也不必擔心失智症患者被一些沒良心的傭人欺負了!
